content/01-contributing-oss/slides/index.en.md

```diff
@@ -0,0 +1,162 @@
+---
+outputs:
+- Reveal
+title: Introductory slides
+hidden: true
+layout: list
+weight: 1
+output: hugodown::md_document
+rmd_hash: f0da839050cdbe81
+
+---
+
+Contributing to Open Source Software
+====================================
+
+:wave: Welcome!
+
+------------------------------------------------------------------------
+
+I'm Bob, I like science, people, code and records.
+
+------------------------------------------------------------------------
+
+<img src="/images/hackfest2020.PNG" height=200/>
+
+- A one month celebration of Open Source on GitHub.
+
+- Meaningful contributions to the open source ecosystem.
+
+- *4* pull requests to a public repository on GitHub to get a t-shirt or a tree (don't need to be accepted).
+
+---
+
+### What is Open Source?
+
+> Software whose source code is released under a license in which the copyright holder grants users the rights to **study, change, and distribute** the software to anyone and for any purpose.
+
+https://choosealicense.com/
+
+---
+
+### Examples
+
+[Linux](https://thelinuxfoundation.org), [Firefox](https://www.mozilla.org/en-GB/firefox/), [R & R packages](https://www.r-project.org/), [python](https://www.python.org/), [Apollo-11](https://github.com/chrislgarry/Apollo-11), [BBC Data Unit](https://github.com/BBC-Data-Unit), [This Talk](https://github.com/RSE-Sheffield/hacktoberfest2020)!
+
+---
+
+### Why open source?
+
+- Open and reproducible research
+- Less need to reinvent
+- Lower costs, maybe
```

Comment:
There are some direct costs that are unarguably lower. An example: renting cloud compute on AWS is more expensive if you choose Windows, a proprietary operating system, over Linux, a Open Source one.

Reply:
Thanks. Really good point. I will also bring up that you don't get support so something looks like it's free but you still need to install, patch, maintain, document...